### PR TITLE
Remove ibm_db references

### PR DESCRIPTION
**Pull request by naveen-ram**

### CODE REVIEW

File: django_ibmi/base.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 """
 DB2 database backend for Django.
-Requires: ibm_db_dbi (http://pypi.python.org/pypi/ibm_db) for python
+Requires: pyodbc for python
```
Comment (kadler) [inline django_ibmi/base.py]:
```suggestion
Requires: pyodbc
```

For pyodbc, Python is implied.

File: django_ibmi/introspection.py [hunk 2]
```diff
@@ -27,34 +27,51 @@
...
+        pyodbc.SQL_TYPE_DATE: 'DateField',
+        pyodbc.SQL_TYPE_TIME: 'TimeField',
+        pyodbc.SQL_TYPE_TIMESTAMP: 'DateTimeField',
+        pyodbc.SQL_VARBINARY: 'BinaryField',
+        pyodbc.SQL_VARCHAR: 'TextField',
+        pyodbc.SQL_LONGVARCHAR: 'TextField',
+        pyodbc.SQL_WCHAR: 'CharField',
+        pyodbc.SQL_WLONGVARCHAR: 'TextField',
+        pyodbc.SQL_WVARCHAR: 'TextField',
+        pyodbc.SQL_SS_XML: 'XMLField'
```
Comment (kadler) [inline django_ibmi/introspection.py]:
```suggestion
```
Pretty sure SQL_SS_XML is a Microsoft SQL Server specific datatype

File: django_ibmi/introspection.py [hunk 3]
```diff
@@ -27,34 +27,51 @@
...
+# after fix to DatabaseWrapper and CursorWrapper
+
 class DatabaseIntrospection( BaseDatabaseIntrospection ):
     
     """
     This is the class where database metadata information can be generated.
     """
 
+    SQL_AUTOFIELD = -777555
+    SQL_BIGAUTOFIELD = -777444
```
Comment (kadler) [inline django_ibmi/introspection.py]: What are these for?

Comment (naveen-ram) [reply django_ibmi/introspection.py]: I believe I pulled these from this repo https://github.com/lionheart/django-pyodbc/blob/master/django_pyodbc/introspection.py so it is most likely defining Microsoft SQL Server datatypes. For now I removed the whole reverse data types change and just put a TODO to handle that in a separate PR.

File: django_ibmi/introspection.py (hunk 2 above)
Comment (kadler) [reply django_ibmi/introspection.py]: The standard ODBC types looked right I think. I was just referring to SQL_SS_XML.

File: django_ibmi/introspection.py (hunk 3 above)
Comment (kadler) [reply django_ibmi/introspection.py]: Ok.

**Review by kadler — APPROVED** [review]

**Pull request merged**